mate-panel/panel.c

```diff
@@ -598,6 +598,109 @@ drop_caja_desktop_uri (PanelWidget *panel,
 	return success;
 }
 
+enum {
+	RESPONSE_SET_BACKGROUND,
+	RESPONSE_CREATE_LAUNCHER
+};
+
+static void
+image_drop_dialog_response (GtkWidget   *dialog,
+			    int          response,
+			    PanelWidget *panel)
```

Comment:
indent

Reply:
@rbuj  the indentation is ok, see https://github.com/mate-desktop/mate-panel/blob/eff98bfc06d0e55f9c375fe9ebd48328b5e1e913/mate-panel/panel.c

it respects old style